App/src/screen/Login/Login.js

```diff
@@ -17,6 +19,16 @@ const Login = ({navigation}) => {
   const login = () => {
     console.log(state);
   };
+
+  const formik = useFormik({
+    initialValues: initialValues(),
+    validationSchema: Yup.object(validationSchema()),
+    validateOnChange: false,
+    onSubmit: formValue => {
+      console.log('Formulario enviado..');
+      console.log(formValue);
```

Comment:
Delete the console.log() of your code in GitHub, you can use Alert.alert()

Reply:
solved problems